janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLStoreManager.java

```diff
@@ -231,6 +231,8 @@ public CQLStoreManager(final Configuration configuration) throws BackendExceptio
                 "server, please check %s and %s options", PARTITIONER_NAME.getName(), METADATA_TOKEN_MAP_ENABLED.getName()));
         }
         switch (partitioner) {
+            case "DefaultPartitioner": // Amazon managed KeySpace uses com.amazonaws.cassandra.DefaultPartitioner
+                fb.timestamps(false).cellTTL(false);
```

Comment:
I have no idea how we could possibly test this. If anyone has an idea, please let me know.

Reply:
I guess in this situation it should be tested manually because we don't have access to Amazon managed cassandra to test it. Also, I didn't find Docker containers which mimic Amazon managed Cassandra. 
We could also refactor the code to follow DI principle and move cql session creation and store features creation to separate builders / factories. If so, we can mock session and cover store features creation by unit tests but I wouldn't do it in the same PR.
Thus, I think that's fine to not cover this branch with tests in this PR